doc/release-notes-26265.md

```diff
@@ -0,0 +1,6 @@
+P2P and network changes
+---------
+
+- Transactions of non-witness size 65 and above are now allowed by mempool
```

Comment:
Suggest adding the unit and perhaps also stating what the previous value was to clarify the change.

```suggestion
- Transactions of non-witness size 65 bytes and above are now allowed by mempool
```

Reply:
will add if I ever touch the PR